src/STAT.md

```diff
@@ -42,34 +42,34 @@ Mode 0  ___000___000___000___000___000___000________________000
 Mode 1  ____________________________________11111111111111_____
 ```
 
-When the PPU is reading a particular part of video memory,
-that memory is inaccessible to the CPU.
+When the PPU is accessing some video-related memory, that memory is inaccessible
+to the CPU: writes are ignored, and reads return garbage values (usually $FF).
```

Comment:
```suggestion
to the CPU: writes are ignored, and reads return garbage values (usually $FF).
```
Why is there a comma before and here?

Reply:
Because I found its lack makes the sentence harder to process.